codelists/tests/test_actions.py

```diff
@@ -425,6 +427,96 @@ def test_export_to_builder(organisation_user, new_style_version):
     assert draft.searches.count() == new_style_version.searches.count()
 
 
+def test_update_draft(organisation_user, version_with_complete_searches):
```

Comment:
Hmm. It's callled `test_update_draft` because that's what I was initially going to test, and then I realised that it was tricky to mock `_recreate_draft_searches` to pretend it had returned new concepts, because the test fixtures use `export_to_builder`, which needs it to do the right thing.  I can rename to `test_check_new_concepts` (I don't suppose the `__` is necessary